src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -50,6 +48,41 @@
 #include "utils/string_view.h"
 #include "utils/thread_access_checker.h"
 
+METRIC_DEFINE_counter(replica,
+                      bulk_load_download_count,
```

Comment:
Like `bulk_load_running_count`, how about rename the following two metrics to `bulk_load_downloading_count` and `bulk_load_ingesting_count` ?

And in fact they are gauge type?

Reply:
OK. Actually in perf counter they are accumulative counter. We could observe their increase. We could also change them to gauge, by decreasing them while finished. Is this OK ?